Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml

```diff
@@ -22,7 +22,7 @@
   id: ScarTopSurgeryShort
   bodyPart: Chest
   markingCategory: Chest
-  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi, Vulpkanin] # DeltaV - Felinid-Vulpkanin
+  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin, Kitsune]  # DeltaV - Felinid-Vulpkanin-Kitsune
```

Comment:
see above
Also you added Harpy and Rodentia. Was this discussed or is this a copy-paste mishap?